third_party/tensorrt/plugin/BUILD

```diff
@@ -0,0 +1,60 @@
+# NVIDIA TensorRT Open Source Plugins
+# This package contains build targets for select TensorRT plugins included in the
+# TensorRT open source repository.
+load("@local_config_cuda//cuda:build_defs.bzl", "cuda_default_copts", "cuda_library")
+
+exports_files(["LICENSE"])
+
+cuda_library(
+    name = "plugin_common",
+    srcs = [
+        "plugin/common/kernels/common.cu.cc",
```

Comment:
Does this actually work in OSS build? Since the BUILD file is inside plugin, shall we take off "plugin" from this line?
Similar comment for the rest of this files, including the hdrs.

Reply:
"plugin/..." refers to path in TRT OSS directory, which is downloaded in the Workpace.bzl file. Yes it does work, I've tested with the test case pushed up in the second commit here